blocks/color-palette/index.js

```diff
@@ -61,15 +61,12 @@ function ColorPalette( { colors, value, onChange } ) {
 				) }
 			/>
 
-			<div className="blocks-color-palette__item-wrapper blocks-color-palette__clear-color">
-				<button
-					className="blocks-color-palette__item"
-					onClick={ () => onChange( undefined ) }
-					aria-label={ __( 'Remove color' ) }
-				>
-					<span className="blocks-color-palette__clear-color-line" />
-				</button>
-			</div>
+			<button className="button-link blocks-color-palette__clear"
```

Comment:
For consistency, and in the case this component gets reused within a form element, I'd consider to add a `type="button"`

Reply:
I also think this could be consolidated into the Button component, similar to how we have `<Button isPrimary />`, e.g. `<Button isLink />` to handle the styling and sensible defaults.

Reply:
Hi @aduth, unfortunately, color palette does not use button component I did a simple test to use it and in other cases, the design looks different. Also adding a setting isTypeButton to button component would break some tests. I will try to address this feedback in a separate PR, that makes color palette uses our button component.